src/js/trove/parse-pyret.js

```diff
@@ -239,11 +239,11 @@
         },
         'spy-field': function(node) {
           if (node.kids.length === 1) {
-            return RUNTIME.getField(ast, 's-spy-name')
-              .app(pos(node.pos), tr(node.kids[0]));
+            return RUNTIME.getField(ast, 's-spy-expr')
+              .app(pos(node.pos), RUNTIME.makeString(""), tr(node.kids[0]), RUNTIME.makeBoolean(true));
```

Comment:
Here, you know (because of the grammar) that `node.kids[0]` is an identifier, so I think the right thing to do here is `string(node.kids[0])`